src/main/java/org/opentripplanner/graph_builder/module/geometry/GeometryAndBlockProcessor.java

```diff
@@ -186,7 +183,7 @@ public void run(Graph graph, DataImportIssueStore issueStore) {
     private void interline(Collection<TripPattern> tripPatterns, Graph graph) {
 
         /* Record which Pattern each interlined TripTimes belongs to. */
-        Map<TripTimes, TripPattern> patternForTripTimes = Maps.newHashMap();
+        Map<TripTimes, TripPattern> patternForTripTimes = new ConcurrentHashMap<>();
```

Comment:
@leonardehrenfried Can you enlighten me? I do not see any concurrency going on in this classes. We can use regular `HashMap`s if so is the case. (This comment apply to this method, as well as the class map fields).